RELEASE.md

```diff
@@ -0,0 +1,99 @@
+# Release Checklist
+
+The following checks should be completed before publishing a new release of the
+Moonbeam runtime or client.
+
+### Runtime Releases
+
+The following should be done *prior* to tagging the potential release. Upon
+completion, tag the commit and proceed with the [All Releases](#all-releases) steps.
+
+- [ ] List any [native runtime](#native-runtimes) versions associated with the release.
+- [ ] Has incremented  [`spec_version`](#spec-version) for any native runtimes from any existing use on public (non-private/test) networks.
+- [ ] Verify [new migrations](#new-migrations) complete successfully, and the runtime state is
+  correctly updated.
+- [ ] Verify previously [completed migrations](#old-migrations-removed) are removed.
+- [ ] Verify pallet and [extrinsic ordering](#extrinsic-ordering) has stayed the same. Bump
+  `transaction_version` if not.
+- [ ] Verify new extrinsics have been correctly whitelisted/blacklisted for
+  [proxy filters](#proxy-filtering).
+- [ ] Verify [benchmarks](#benchmarks) have been updated for any modified runtime logic.
+- [ ] Verify [Polkadot JS API](#polkadot-js) are up to date with the latest runtime changes.
+
+### All Releases
+
+- [ ] Check that the new client versions have [run on the network](#burn-in) without issue for 12
+  hours.
+- [ ] Check that a draft release has been created at https://github.com/PureStake/moonbeam/releases with relevant [release notes](#release-notes)
+- [ ] Check that [build artifacts](#build-artifacts) have been added to the draft-release
+
+## Notes
+
+### Burn In
+
+Ensure that DevOps has run the new release on Moonbeam TestNet validators for
+at least 12 hours prior to publishing the release.
+
+### Build Artifacts
+
+Add any necessary assets to the release. They should include:
+
+- Linux binary
+- GPG signature of the Linux binary
+- SHA256 of binary
+- Source code
+- Wasm binaries of any runtimes
+
+### Release notes
+
+The release notes should list:
+
+- The priority of the release (i.e., how quickly users should upgrade)
+- Which native runtimes and their versions are included
+- The proposal hashes of the runtimes as built with [srtool](https://gitlab.com/chevdor/srtool)
+
+The release notes may also list:
+
+- Free text at the beginning of the notes mentioning anything important regarding this release
+- Notable changes (those labelled with B[1-9]-* labels) separated into sections
```

Comment:
These are Parity-specific (unless we're planning to do the same thing).

Reply:
Yes, I was hoping to use the same (it also allow us to add the ones from substrate too) but it might be too much.